dhis-2/dhis-api/src/main/java/org/hisp/dhis/program/Enrollment.java

```diff
@@ -333,12 +333,12 @@ public void setMessageConversations(List<MessageConversation> messageConversatio
   @JsonProperty("trackedEntityComments")
```

Comment:
Should we change this @JsonProperty to be `notes`?

Reply:
@enricocolasante who relies on this `JsonProperty`? New tracker certainly is not as it uses the view classes https://github.com/dhis2/dhis2-core/blob/e52845ef22af62153888570a8aeaa49bb9640cdc/dhis-2/dhis-web-api/src/main/java/org/hisp/dhis/webapi/controller/tracker/view/Enrollment.java#L100